include/ebpf_program_attach_type_guids.h

```diff
@@ -57,6 +68,17 @@ extern "C"
     __declspec(selectany) ebpf_attach_type_t EBPF_ATTACH_TYPE_CGROUP_INET6_CONNECT =
         EBPF_ATTACH_TYPE_CGROUP_INET6_CONNECT_GUID;
 
+#define EBPF_ATTACH_TYPE_CGROUP_INET6_AUTH_CONNECT_GUID                                \
+    {                                                                                  \
+        0x54b0b6ed, 0x432a, 0x4674, { 0x8b, 0x27, 0x8d, 0x9f, 0x5b, 0x40, 0xc6, 0x75 } \
+    }
+    /**
+     * @brief The program attached to the INET6_AUTH_CONNECT hook will be invoked for
+     * connect() calls on TCP or UDP sockets before the connection authorized.
```

Comment:
```suggestion
     * connect() calls on TCP or UDP sockets before the connection is authorized.
```
Is this what was intended?